tests/test_run.py

```diff
@@ -90,3 +108,57 @@ def test_main_match(test_directory):
 
     with open(mock_expected_path) as expected:
         assert result_file.read() == expected.read()
+
+
+def test_scraper(monkeypatch, test_directory):
+    def mock_get_mapping(_):
+        return [
+            {'property': 'P321', 'url': 'http://schema.org/director'},
+            {'property': 'P123', 'url': 'http://schema.org/name'},
+        ]
+
+    def mock_normalize_from_extruct(_, data):
+        return [{
+            "http://schema.org/director": [
+                data['microdata'][0]['director']['name']
+            ],
+            "http://schema.org/genre": [
+                data['microdata'][0]['genre']
+            ]
+        }]
+
+    monkeypatch.setattr(SchemaOrgNormalizer, "normalize_from_extruct", mock_normalize_from_extruct)
+    monkeypatch.setattr(WdqsSchemaorgPropertyMapper, "get_mapping", mock_get_mapping)
+    monkeypatch.setattr(requests, "Session", MockSession)
```

Comment:
Since we are using these in several places now, I think creating a set of reusable fixtures will make more sense.

Reply:
hmm, let me see what I can do about it.

Reply:
```
ImportError while loading conftest '/home/amsa/workspace/ref2/tests/conftest.py'.
tests/conftest.py:15: in <module>
    @pytest.fixture('package')
E   ValueError: class fixtures not supported (maybe in the future)
```
:(

Reply:
https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions

Reply:
Yeah, looked at it and finally managed to move some fixture into conftest.py and then I will clean the rest of codebase after this is in to avoid making this too big.